rfcs/026-relevance_reporting_service/README.md

```diff
@@ -0,0 +1,123 @@
+# RFC 026: Relevance reporting service
+
+**Last updated: 06 July 2020.**
+
+## Glossary
+
+* **elastic-query:** A blob of JSON we send to Elasticsearch
+* **search-terms:** A set of terms sent as a string from a client to a search service
+
+## Summary
+
+A way for us to be able to test and report on the efficacy of elastic-queries by being able to compare a set of search-terms and their respective expected results and ordering.
+
+## Motivation
+
+We need to have and instil transparency and confidence in the way that our search relevancy works.
+
+Any test to help with this will need to be against our production data, as that is exactly what people will be seeing.
+
+### Use cases
+
+* Developers and data-scientists being able to write new search queries with  confidence in improvement and without fear of regression
+* Allowing internal teams to have a greater understanding and more input as to how the search works, thus being able to share this with external researchers
+* External researchers interested in how their results are being returned
+
+## Proposal
+
+![architecture](./relevance_service.png)
+
+1. Creating a public API where you can send elastic-queries
```

Comment:
Why should the API be public?

Reply:
```suggestion
1. Creating an API where you can send elastic-queries
```

Was more about being accessible, but thinking about it - that's the nature of an API, public or not.